mock/py/mockbuild/backend.py

```diff
@@ -203,6 +204,18 @@ def installSrpmDeps(self, *srpms):
         finally:
             self.uid_manager.restorePrivs()
 
+    @traceLog()
+    def installSpecDeps(self, spec_file):
+        try:
+            spec=rpm.spec(spec_file).sourceHeader.dsFromHeader()
+            self.uid_manager.becomeUser(0, 0)
+            for i in range(len(spec)): 
+                requirement_name = spec[i].split(" ")[1]
```

Comment:
I did not test this yet, but this will likely not work for something like `BuildRequires: mock-core-configs > 26`. Can you try it?

Reply:
It will ignore everything, after name of the package.